tests/pykafka/test_balancedconsumer.py

```diff
@@ -253,8 +253,8 @@ def test_rebalance_unblock_event(self):
         consumer_b_thread = threading.Thread(target=consumer_b.consume)
         consumer_b_thread.start()
 
-        consumer_a_thread.join(10)
-        consumer_b_thread.join(10)
+        consumer_a_thread.join(30)
```

Comment:
@emmett9001 What timeout would you consider appropriate for thread join? I kept it sufficiently large to be on the safer side, but at the cost of increase Travis build times.

Reply:
I'd try to keep it as short as possible, 10s is probably a good starting point.